<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ß—Ç–æ —è —Ö–æ—á—É –ø–æ—Å–º–æ—Ç—Ä–µ—Ç—å</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_1">
    <item>
     <widget class="QPushButton" name="btnToggleTheme">
      <property name="text">
       <string notr="true">üåô</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>–í—ã—à–µ–¥—à–µ–µ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutReleased">
        <item>
         <layout class="QHBoxLayout" name="categoryLayout">
          <item>
           <widget class="QComboBox" name="comboCategoryReleased"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddCategory">
            <property name="text">
             <string>–ù–æ–≤–∞—è –∫–∞—Ç–µ–≥–æ—Ä–∏—è</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveCategory">
            <property name="text">
             <string>–£–¥–∞–ª–∏—Ç—å –∫–∞—Ç–µ–≥–æ—Ä–∏—é</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listReleased"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="releasedButtons">
          <item>
           <widget class="QPushButton" name="btnAddReleased">
            <property name="text">
             <string>–î–æ–±–∞–≤–∏—Ç—å</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditReleased">
            <property name="text">
             <string>–†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞—Ç—å</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveReleased">
            <property name="text">
             <string>–£–¥–∞–ª–∏—Ç—å</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>–ù–µ –≤—ã—à–µ–¥—à–µ–µ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutUpcoming">
        <item>
         <widget class="QListView" name="listUpcoming"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="upcomingButtons">
          <item>
           <widget class="QPushButton" name="btnAddUpcoming">
            <property name="text">
             <string>–î–æ–±–∞–≤–∏—Ç—å</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditUpcoming">
            <property name="text">
             <string>–†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞—Ç—å</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveUpcoming">
            <property name="text">
             <string>–£–¥–∞–ª–∏—Ç—å</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>–®–ª–∞–∫</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutTrash">
        <item>
         <widget class="QListView" name="listTrash"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="trashButtons">
          <item>
           <widget class="QPushButton" name="btnAddTrash">
            <property name="text">
             <string>–î–æ–±–∞–≤–∏—Ç—å</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditTrash">
            <property name="text">
             <string>–†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞—Ç—å</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveTrash">
            <property name="text">
             <string>–£–¥–∞–ª–∏—Ç—å</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
